ChromecastSampleApp/ChromecastSampleApp/OOCastManagerFetcher.m

```diff
@@ -11,6 +11,6 @@
 
 @implementation OOCastManagerFetcher
 +(OOCastManager*) fetchCastManager {
-  return [OOCastManager getCastManagerWithAppID:@"D4C7BD4A" namespace:@"urn:x-cast:ooyala"];
+  return [OOCastManager getCastManagerWithAppID:@"B3560E04" namespace:@"urn:x-cast:ooyala"];
```

Comment:
B3560E04 is the default receiver. Do you mean it is better for it to be the custom one? Thanks.